### ISSUE TITLE
Forwarding not working

### ISSUE THREAD

**Issue by rhukster** [beyondcode/helo-community#10]
I'm trying to forward an email to another account, but even though it's authenticating with Google, it seems to have no "to" email address:

![Image 2020-04-22 at 3 12 43 PM](https://user-images.githubusercontent.com/1084697/80034643-fe5cf380-84ab-11ea-9ab9-ad653753af6f.png)

The `from:` and `to:` were both valid email addresses.

BTW, it's unclear what the "Encryption" field should be set to in forwarding settings, I set it to `TLS` and port `587`:

![Image 2020-04-22 at 3 16 52 PM](https://user-images.githubusercontent.com/1084697/80034886-5562c880-84ac-11ea-9d54-b6915ad51a27.png)

**Comment by rhukster** [comment]
Actually got a different error this time.. Looks like it's on my end now.

**Comment by sschlein** [comment]
@rhukster did you get it working with the Google SMTP? If yes, can you tell me your settings? I'd like to add this to the docs 🙂

**Comment by rhukster** [comment]
Well turns out I think it was the two things.  First I was changing the from/to in the forward dialog, but as I mentioned in #11, that was not working. After inspecting the logs, I realized that it was using the email's from/to no matter what I set them two in the input fields.

The second was I had to enable the `smtp-relay` option in the admin of my Google GSuite Domain.  Then I had to configure that to allow the IP of my server, etc, require TLS, etc.
